### ISSUE TITLE
Missing transformations

### ISSUE THREAD

**Issue by adelavega** [neuroscout/neuroscout#719]
There's a are some transformations implemented by pybids we might want to add to the frontend:

- Replace
- ToDense
- Split
- Factor
- Rename
- Group
- Filter
- DropNA
- Delete
- Copy
- Assign
- Sum
- Product

Of these, I would say the most useful ones are probably: `Replace` (for categorical variables, such as Google's, e.g. "VERY UNLIKELY"). `Factor`, `DropNA`, `Product`, `Sum`. 

The main thing to be careful with is that the users might find this step confusing.

**Comment by tyarkoni** [comment]
Re: `Replace`, I agree that this would be a good one, but for it to be useful we'd need to have a way of showing users what string values any categorical columns have, which seems like it might be more trouble than it's worth at this point.

The main issue with `Product` and `Sum` is that they necessarily require a new column output, and I think we were still operating on the assumption that no new columns are created, since that then requires the front end to either have a model of what transformations do, or iteratively communicate with the back end to compute the available variable names (e.g., if you want to do a `Product` followed by a `Standardize`). If we're ready to take that on, great, but it does seem like a bunch of additional complexity.

**Comment by adelavega** [comment]
Yep, agreed on both fronts. About the `Replace`, we may want to disable predictors that are categorical then, because they otherwise are going to impossible to use. We could also preprocess them ourselves (e.g. make them numeric), but this means we have to make potentially strong assumptions for the user.

**Comment by adelavega** [comment]
It would be interesting to try face emotion, and see if its any good though. We could easily do it through `pyNS` for now. 

At least we could warn users in the `Review` tab about categorical variables for now also, rather than disabling (to not prevent `pyNS` usage).
